_dev/src/components/product-feed/settings/delivery-time-and-rates/create-custom-carrier.vue

```diff
@@ -0,0 +1,317 @@
+                        variant="text"
+                        v-b-tooltip:psxMktgWithGoogleApp
+                        :title="''"
+                      >
+                        <span class="material-icons-round text-primary mb-0 ps_gs-fz-16 w-16">
+                          info_outlined
+                        </span>
+                      </b-button>
+                    </div>
+                  </b-col>
+                  <b-col class="col-auto">
+                    <b-input-group
+                      :append="budgetCurrencySymbol"
+                      class="ps_gs-carrier__input-number-group"
+                    >
+                      <b-form-input
+                        type="number"
+                        class="ps_gs-carrier__input-number no-arrows ps_gs-mw-90"
+                        size="sm"
+                        :state="validateAmountRate(carrier[carrier.offerChosen].freeShippingAmount)"
+                        v-model.number="carrier[carrier.offerChosen].freeShippingAmount"
+                      />
+                    </b-input-group>
+                  </b-col>
+                </b-row>
+              </div>
+            </b-card>
+            <!-- eslint-enable max-len -->
+          </b-col>
+        </b-row>
+      </b-container>
+    </b-card>
+  </div>
+</template>
+<script lang="ts">
+import Vue, {PropType} from 'vue';
+import {OfferType} from '@/enums/product-feed/offer';
+import symbols from '@/assets/json/symbols.json';
+import {validateDeliveryTime, CustomCarrier, validateOfferChoice} from '@/providers/shipping-rate-provider';
+
+export default Vue.extend({
+  name: 'ProductFeedSettingsShipping',
+  components: {
+  },
+  props: {
+    carrier: {
+      type: Object as PropType<CustomCarrier>,
+      required: true,
+    },
+    validationError: {
+      type: Boolean,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      OfferType,
+      offers: [
+        {
+          text: this.$t('productFeedSettings.deliveryTimeAndRates.estimateStep.storeOffers.freeShippingRateForAllProducts'),
+          value: OfferType.FLAT_SHIPPING_RATE,
+        },
+        {
+          text: this.$t('productFeedSettings.deliveryTimeAndRates.estimateStep.storeOffers.freeShippingForAllProducts'),
+          value: OfferType.FREE_SHIPPING,
+        },
+        {
+          text: this.$t('productFeedSettings.deliveryTimeAndRates.estimateStep.storeOffers.freeShippingOverAmount'),
+          value: OfferType.FREE_SHIPPING_OVER_AMOUNT,
+        },
+      ],
+    };
+  },
+  computed: {
+    validateTimeDelivery(): boolean|null {
+      return validateDeliveryTime(this.carrier) ? null : false;
+    },
+    validateCarrierName(): boolean|null {
+      if (!this.carrier.carrierName?.length) {
+        return null;
+      }
+      if (this.carrier.carrierName.length <= 90
+        && this.carrier.carrierName.length > 0
+      ) {
+        return true;
+      }
+
+      return false;
```

Comment:
Duplicate of the method of the same name in _dev/src/providers/shipping-rate-provider.ts?